doc: Upgrade guide for v1.31.0 (#3205)
mongodb/terraform-provider-mongodbatlas: docs/guides/1.31.0-upgrade-guide.md

---
page_title: "Upgrade Guide 1.31.0"
---

# MongoDB Atlas Provider 1.31.0: Upgrade and Information Guide

The Terraform MongoDB Atlas Provider version 1.31.0 has a number of new and exciting features.

## New Resources, Data Sources, and Features

> - You can now skip the default alert settings for an organization using `skip_default_alerts_settings` in the `mongodbatlas_organization` resource. To learn more, see the [mongodbatlas_organization](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/organization) resource documentation.

Reviewer: ```suggestion
- You can now manage the default alert settings for an organization using `skip_default_alerts_settings` attribute  in the `mongodbatlas_organization` resource & data sources. To learn more, see the [mongodbatlas_organization](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/organization) resource documentation.
```

> 

Reviewer 2: ```suggestion
- You can now use the [Atlas CLI plugin](https://github.com/mongodb-labs/atlas-cli-plugin-terraform) to generate the `mongodbatlas_advanced_cluster` resource definition from a `mongodbatlas_cluster` definition. See our [migration guide](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/cluster-to-advanced-cluster-migration-guide#migration-using-the-moved-block-recommended) for more information.
```

Reviewer 3: @marcosuma sorry I just saw this, can you see my proposal and tell which one you prefer?

Reviewer 2: ok I'll make some changes directly, sorry for the noise (cc @oarbusi)

---
page_title: "Upgrade Guide 1.31.0"
---

# MongoDB Atlas Provider 1.31.0: Upgrade and Information Guide

The Terraform MongoDB Atlas Provider version 1.31.0 has a number of new and exciting features.

## New Resources, Data Sources, and Features

> - You can now manage the default alert settings for an organization using `skip_default_alerts_settings` attribute  in the `mongodbatlas_organization` resource & data sources. To learn more, see the [mongodbatlas_organization](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/organization) resource documentation.

Reviewer 4: ```suggestion
- You can now optionally skip creating the default set of alerts for new organizations, using the `skip_default_alerts_settings` attribute  in the `mongodbatlas_organization` resource & data sources. To learn more, see the [mongodbatlas_organization](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/organization) resource documentation.
```

Reviewer 4: [nit] Lets add the impact on the customer experience & why the change matters

- You can now use the new [Atlas CLI plugin](https://github.com/mongodb-labs/atlas-cli-plugin-terraform) to help with the migration from `mongodbatlas_cluster` to `mongodbatlas_advanced_cluster`. For detailed instructions, see the [Migration Guide: Cluster to Advanced Cluster](cluster-to-advanced-cluster-migration-guide).

> 1.31.0 also includes general improvements, bug fixes, and several key documentation updates. See the [CHANGELOG](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md) for more information.

Reviewer 4: Is this true & what are the changes? In general I'd prefer if the changes were all includes in the upgrade guide, unless they are truly small/not relevant.

As a customer, the statement as written makes me cautious that unexpected changes could impact my use of the new version.

Reviewer 2: fair point, this is not true for this one so we can remove it
